compiler/nimsuggest/nimsuggest.nim

```diff
@@ -325,11 +325,10 @@ proc handleCmdLine() =
     processCmdLine(passCmd2, "")
     mainCommand()
 
-when false:
-  proc quitCalled() {.noconv.} =
-    writeStackTrace()
+proc quitCalled() {.noconv.} =
+  writeStackTrace()
 
-  addQuitProc(quitCalled)
```

Comment:
Please disable that again. There must be no stdout output in order to not block the socket based server.